package.json

```diff
@@ -8,9 +8,8 @@
     "*.{js,jsx,ts,tsx}": "eslint --cache --fix"
   },
   "scripts": {
-    "ci:publish": "yarn workspaces foreach --all --topological --exclude @openmrs/esm-home npm publish --access public --tag latest",
-    "ci:prepublish": "yarn workspaces foreach --all --topological --exclude @openmrs/esm-home npm publish --access public --tag next",
-    "ci:release": "yarn workspaces foreach --all --topological version",
+    "ci:publish": "yarn npm publish --access public",
```

Comment:
Shouldn't we exclude `@openmrs/esm-home` (monorepo)?
We just want to publish `@openmrs/esm-home-app` (app)

Reply:
Oops! my bad. Didn't realise that home-app is under the `packages` directory